include/lsst/afw/fits.h

```diff
@@ -312,7 +312,7 @@ class Fits {
     std::string getFileName() const;
 
     /// Return the current HDU (0-indexed; 0 is the Primary HDU).
-    int getHdu();
+    int getHdu() const;
```

Comment:
I don't think this is conceptually correct even if it compiles; the implementation calls a C library function via a pointer that can change state that we want to consider part of the object.